etna/ciim/constants.py

```diff
@@ -102,7 +102,6 @@ def __iter__(self):
         Bucket(key="blog", label="Blog posts"),
         Bucket(key="researchGuide", label="Research Guides"),
         Bucket(key=BucketKeys.INSIGHT.value, label="Insights"),
-        Bucket(key=BucketKeys.HIGHLIGHT.value, label="Highlights"),
         Bucket(key="audio", label="Audio"),
```

Comment:
There is some code around website highlight [WebsiteSearchView](https://github.com/nationalarchives/ds-wagtail/blob/9d7bdb78edb9000adcc67bdc676102e9decc001a/etna/search/views.py#L723-L724),  and unit tests are skipped
Would it be Ok to then comment out those bits which are removed?

Reply:
Yes, that's a good shout. Implemented in [c92c7a2](https://github.com/nationalarchives/ds-wagtail/pull/775/commits/c92c7a272164a005c395fe7a5d91123ed25e10e7)